src/covid_model_seiir_pipeline/lib/ihme_deps.py

```diff
@@ -0,0 +1,35 @@
+"""Wrappers for IHME specific dependencies.
+
+This module explicitly declares and wraps IHME specific code bases
+to prevent CI failures at import time.
+"""
+
+
+def _lazy_import_error_object(module_name: str):
+
+    class C:
+        def __init__(self, *args, **kwargs):
+            raise ModuleNotFoundError(f"No module named '{module_name}'")
+
+
+def _lazy_import_error_function(module_name: str):
+    def f(*args, **kwargs):
+        raise ModuleNotFoundError(f"No module named '{module_name}'")
+
```

Comment:
You're not returning anything in either of your `_lazy` methods

Reply:
Very good catch.